master/buildbot/scripts/start.py

```diff
@@ -56,6 +57,10 @@ def _failure(self, why):
 file during startup. Please inspect and fix master.cfg, then restart the
 buildmaster.
 """)
+        elif why.check(BuildmasterStartupError):
+            print("""
+The buildmaster startup failed. Please see 'twistd.log' for possible reason.
```

Comment:
@tardyp how about I'll use `textwrap.dedent` in a following pull request? It's worth to use it not only in my changes, but in other parts of the code, which I don't want to mix with in this PR that fixes master startup failure handling.